client/bench.go

```diff
@@ -0,0 +1,214 @@
+package bench
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	matmult "github.com/amwolff/2-lg-mats-bench/gen/go/amwolff/matmult/v1"
+	"gonum.org/v1/gonum/mat"
+	"time"
+)
+
+const (
+	serverAddr = "0.0.0.0:50051"
+	pbPath     = "/tmp/MATRIX"
+)
+
+type Writer interface {
+	writeRequest() matmult.MultiplyRequest
+}
+
+type Parser interface {
+	parseRequest(request matmult.MultiplyRequest)
+	parseResponse(response matmult.MultiplyResponse)
+}
+
+// The data is stored in column-major order
+type Matrix struct {
+	Data    []float64
+	Columns int
+	Rows    int
+}
+
+type goNumMessage struct {
+	Multiplier   mat.Matrix
+	Multiplicand mat.Matrix
+	Product      mat.Dense
+}
+
+type primitiveMessage struct {
+	Multiplier   *Matrix
+	Multiplicand *Matrix
+	Product      *Matrix
+}
+
+func (m Matrix) getValueAt(row int, col int) float64 {
```

Comment:
Maybe
```suggestion
func (m Matrix) coeff(row int, col int) float64 {
```
following Eigen's convention?